### ISSUE TITLE
COREDIR and PLUGINDIR should default to a value derived from PREFIX

### ISSUE THREAD

**Issue by clbr** [mupen64plus/mupen64plus-ui-console#50]
As a new user, I only set PREFIX for all the mupen64 builds. This placed everything in the right paths, but I needed to put full plugin paths to the .cfg file, and start the console ui with a --corelib arg. It would be useful if the core and plugin paths had defaults derived from PREFIX, the same defaults that the parts get placed in.

**Comment by clbr** [comment]
Shouldn't PLUGINDIR be PLUGINDIR := $(LIBDIR)/mupen64plus ? That's from one of the plugin makefiles. When I put just the libdir in the cfg file, it doesn't find the plugins.

Thanks for the quick fixes, building this now.

**Comment by clbr** [comment]
Yes, confirming that COREDIR is fixed, but PLUGINDIR doesn't work. I can't reopen the issue for some reason, github is bork.

**Comment by richard42** [comment]
sorry for the bug, and the github problem. I just pushed another change to the makefile; it should be good now
